pynxtools_mpes/mappings.py

```diff
@@ -0,0 +1,64 @@
+"""
+Mapping dictionaries for the MPES conversion.
+"""
+
+DEFAULT_UNITS = {
+    "X": "step",
+    "Y": "step",
+    "t": "step",
+    "tofVoltage": "V",
+    "extractorVoltage": "V",
+    "extractorCurrent": "A",
+    "cryoTemperature": "K",
+    "sampleTemperature": "K",
+    "dldTimeBinSize": "ns",
+    "delay": "ps",
+    "timeStamp": "s",
+    "energy": "eV",
+    "kx": "1/A",
+    "ky": "1/A",
+}
```

Comment:
I would suggest to eventually remove these. They don't belong here I would say. Either they could be set through the config file, or through the xarray. If they are not provided, there should be no units.

Reply:
Addressed in PR #25